helm/opendistro-es/user-roles-values.yaml

```diff
@@ -0,0 +1,227 @@
+                http_enabled: true
+                transport_enabled: true
+                order: 0
+                http_authenticator:
+                  type: basic
+                  challenge: false
+                authentication_backend:
+                  type: internal
+
+      #Define users, roles, mappings and tenants below
+      internal_users.yml: |-
+        _meta:
+          type: "internalusers"
+          config_version: 2
+
+        #define admin user
+        admin:
+          hash: "$2a$12$VcCDgh2NDk07JGN0rjGbM.Ad41qVR/YFJcgHp0UGns5JDymv..TOG"
+          reserved: true
+          backend_roles:
+          - "admin"
+          description: "Demo admin user"
+
+        #define user for  for the Kibana server
+        kibana:
+          hash: "$2y$12$efAwiXyV.uRLnYy9c0Kdju4hrr8fso2cKC7ObkHTrTk4jGXHlzmzi"
+          reserved: true
+          description: "kibana server user"
+
+        #define user for fluentd
+        fluentd:
+          hash: "$2a$12$u1ShR4l4uBS3Uv59Pa2y5.1uQuZBrZtmNfqB3iM/.jL0XoV9sghS2"
+          reserved: true
+          description: "fluentd user"
+
+        #define user for snapshots
+        snapshot:
+          hash: "$2y$12$HARtGIKFsoVT8ypHHA6QwOx6XPoP.En/ReIjLGI90rVqLHf2Lcmhe"
+          reserved: true
+          description: "manage_snapshot user"
+
+        #define user for ilm
+        ilm:
+          hash: "$2y$12$SHMJ3.DOHJFCVi6wOrLfh.gXcYAgh4dBtRitA0d/Txd2Z3tbSiGoy"
+          reserved: true
+          description: "ilm user"
+
+      roles.yml: |-
+        _meta:
+          type: "roles"
+          config_version: 2
+
+        #roles for fluentd user
+        fluentd:
+          reserved: true
+          cluster_permissions:
+          - "cluster_composite_ops"
+          - "cluster_monitor"
+          - "cluster:admin/ingest/pipe/get"
+          - "cluster:admin/ingest/pipeline/put"
+          - "indices:admin/template/get"
+          - "indices:admin/template/put"
+          index_permissions:
+          - index_patterns:
+            - "kubernetes*"
+            allowed_actions:
+            - "crud"
+            - "create_index"
+          - index_patterns:
+            - "kubeaudit*"
+            allowed_actions:
+            - "crud"
+            - "create_index"
+          - index_patterns:
+            - "other*"
+            allowed_actions:
+            - "crud"
+            - "create_index"
+
+        #roles for ilm user
+        manage_ilm:
+          reserved: true
+          cluster_permissions:
+          - "ODS_CLUSTER_MANAGE_ILM"
+          index_permissions:
+          - index_patterns:
+            - "*"
+            allowed_actions:
+           - "cluster:admin/ilm/*"
+
+      roles_mapping.yml: |-
+        _meta:
+          type: "rolesmapping"
+          config_version: 2
+
+        # user-roles mapping
+        all_access:
+          reserved: true
+          backend_roles:
+          - "admin"
+          description: "Maps admin to all_access"
```

Comment:
What does this do? I see that the `admin` user also has `admin` in `backend_roles` but I don't see how the `all_access` role mapping(?) fits into this?

Reply:
The `all_access` role is already defined in opendistro. I basically used what was there before without any change except the password.
 https://github.com/opendistro-for-elasticsearch/security/blob/f91286f6dcf130d6fdeec34d6d7f2603de832157/src/main/resources/static_config/static_roles.yml#L6